ballerina/Ballerina.toml

```
@@ -0,0 +1,13 @@
+[package]
+org = "ballerina"
+name = "mcp"
+version = "1.13.1"
```

```suggestion
version = "0.8.0"
```
Let's change it in properties.gradle